### PR TITLE
Deleting a droplet

### PR DESCRIPTION
**Pull request by Pritom14**

### CODE REVIEW

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java [hunk 1]
```diff
@@ -87,8 +90,34 @@ public boolean onCreateOptionsMenu(Menu menu) {
...
+            new MaterialDialog.Builder(this)
+                    .title(R.string.delete_droplet)
+                    .content("")
+                    .positiveText("Confirm").onPositive(new MaterialDialog.SingleButtonCallback() {
+                @Override
+                public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which) {
+                    doaClient.performAction(droplet.getId(),ActionType.DESTROY,null).enqueue(new Callback<Action>() {
+                        @Override
+                        public void onResponse(Call<Action> call, Response<Action> response) {
+                            Log.d("DESTROY",response.code()+"");
```
Comment (championswimmer) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java]: Do not convert to string like this 
Use explicit typecast like String.valueOf

**Review by championswimmer — CHANGES_REQUESTED** [review]

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java (hunk 1 above)
Comment (Pritom14) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java]: Okay, but can you please elaborate the changes I have to make, as to where should I use String.valueOf?

**Comment by Pritom14** [comment]
Please review the PR

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java [hunk 2]
```diff
@@ -87,8 +90,34 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
         if (id == R.id.delete_droplet) {
-
             // TODO: 26/11/16 perform delete
+            new MaterialDialog.Builder(this)
+                    .title(R.string.delete_droplet)
+                    .content("")
```
Comment (championswimmer) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java:96]: Add some information
(Maybe some warning like - "This action is not reversible" in the content

**Review by championswimmer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**